schoolopy/main.py

```diff
@@ -129,22 +143,90 @@ def create_building(self, school_id, building):
         """
         return Building(self._post('schools/%s/buildings' % school_id, building.json()))
 
-    def get_users(self):
+    def get_me(self):
+        """
+        Get data of the client using this method (Yourself)
+
+        :return: User object obtained from API. (Of yourself)
+        """
+        return User(self._get('users/me'))
+
+    def get_users(self, inactive=False):
```

Comment:
Initially I was thinking we should make a separate method, but I actually like this a lot more.